easybuild/easyconfigs/v/VASP/VASP-5.4.4-CrayIntel-17.08-cuda-8.0.makefile.include

```diff
@@ -0,0 +1,74 @@
+# Precompiler options
+CPP_OPTIONS= -DHOST=\"LinuxIFC\"\
+             -DMPI -DMPI_BLOCK=8000 \
+             -Duse_collective \
+             -DscaLAPACK \
+             -DCACHE_SIZE=4000 \
+             -Davoidalloc \
+             -Duse_bse_te \
+             -Dtbdyn \
+             -Duse_shmem
+
+CPP        = fpp -f_com=no -free -w0  $*$(FUFFIX) $*$(SUFFIX) $(CPP_OPTIONS)
+
+FC         = ftn
+FCL        = ftn
+
+FREE       = -free -names lowercase
+
+FFLAGS     = -assume byterecl -w
+OFLAG      = -O2
+OFLAG_IN   = $(OFLAG)
+DEBUG      = -O0
+
+BLAS       = $(MKLROOT)/lib/intel64/libmkl_blas95_lp64.a
+LAPACK     = $(MKLROOT)/lib/intel64/libmkl_lapack95_lp64.a
+BLACS      = -lmkl_blacs_intelmpi_lp64
+SCALAPACK  = -L$(MKLROOT)/lib/intel64 -lmkl_scalapack_lp64 -lmkl_intel_lp64 -lmkl_sequential -lmkl_core -lmkl_blacs_intelmpi_lp64
+
+OBJECTS    = fftmpiw.o fftmpi_map.o fft3dlib.o fftw3d.o
+FFT_LIB    = -L$(FFTW_DIR) -lfftw3                                           
+INCS       = -I$(FFTW_INC)           
+LLIBS      = $(SCALAPACK) $(BLACS) $(LAPACK) $(BLAS) $(FFT_LIB)
+
+OBJECTS_O1 += fftw3d.o fftmpi.o fftmpiw.o
+OBJECTS_O2 += fft3dlib.o
+
+# For what used to be vasp.5.lib
+CPP_LIB    = $(CPP)
+FC_LIB     = $(FC)
+CC_LIB     = cc
+CFLAGS_LIB = -O
+FFLAGS_LIB = -O1
```

Comment:
Is it what you really want `-O1`?

Reply:
These are the optimization levels suggested by VASP developers: in general I try to change as little as possible the `makefile.include` that comes with the package, because higher optimization levels need to be tested thouroughly